README.md

```diff
@@ -153,23 +153,31 @@ into a usable state.
 
 ## Testing
 
-- To run backend tests, you'll need to have an instance of Postgres running.
-- When initially running tests, a `dds_test` database will automatically be created in Postgres and migrations will be
-  run on it.
-- After the initial run, tests will not run migrations by default, for the sake of speed. If you have migrations you
-  want to run, you can pass the `--clean` flag.
-- The test database is cleared **_before_** each test, meaning you can potentially run a single test and look at the
-  database afterward to help debug issues.
-
-```bash
-yarn test
-````
+- To run backend IT tests, you\'ll need to have an instance of Postgres running.
+- When initially running IT tests, a `dds_test` database will automatically be created in Postgres and migrations will be run on it.
+- After the initial run, tests will not run migrations by default, for the sake of speed. If you have migrations you want to run, you can pass the `--clean` flag.
+- The test database is cleared **_before_** each test, meaning you can potentially run a single test and look at the database afterward to help debug issues.
+- Optionally specify the workspace for the test (`shared`, `server`).  Default is `server`.
 
 ### Options
-`--debug` - Prints all logs rather than only 'test' level logs.
 
-`--clean` - Drops, recreates, and migrates the test database before running tests.
+- `--debug` - Prints all logs rather than only 'test' level logs.
+
+- `--clean` - Server IT only.  Drops, recreates, and migrates the test database before running tests.
 
+- `--it | --spec` - Optional.  Limits the test to only run for IT or unit test.
```

Comment:
I get this error: 

yarn test --spec 
yarn run v1.22.10
$ ./test.sh --spec
$ yarn workspace @covid19-reports/server test --spec
$ ./test.sh --spec
Warning: Cannot find any files matching pattern "**/*.spec.ts"

There is this "spec" test: packages/shared/src/utils/days.spec.ts

Reply:
I can clarify in the documentation -- without specifying a workspace, test runs in server only by default.  Or, should the test iterate through each workspace?

Reply:
1. I see...Run it  with --spec, it gives the same error
2. I think `yarn test` should run all tests, not just the server tests

Reply:
Defaults to run all workspaces and can still use --it and --spec flags.  README updated.